build.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -x
+set -o errexit
+set -o pipefail
+
+# Get a clean checkout
+git fetch origin || true
+git checkout origin/master || true
+git reset HEAD || true
+git checkout -- || true
+
+# Resolve pending metadata
+./main
+
+# Create a PR
+git checkout -b resolve-daily-conflict
+git add .
+git commit -m 'Resolve daily merge conflicts'
+git push
```

Comment:
@foolip Do you know how to set credentials in a cron job? I want to create a branch and a PR automatically here?

Reply:
maybe it should create a PR, stop auto-merge and assign a reviewer?

Reply:
https://github.com/Ecosystem-Infra/wpt-results/blob/main/.github/workflows/results.yml has an example of how to use a token to push to a branch.

But since the intention is to create a PR, I would recommend using `peter-evans/create-pull-request` which handles lots of the boring details. https://github.com/web-platform-tests/wpt/blob/master/.github/workflows/interfaces.yml has an example of that.

Reply:
with `peter-evans/create-pull-request`, I can delete build.sh